user/user_app.go

```diff
@@ -0,0 +1,68 @@
+package user
+
+type UserAppID string
+type AccountID string
+type AllowlistType string
+
+const (
+	AllowlistTypeContracts  AllowlistType = "contracts"
+	AllowlistTypeMethods    AllowlistType = "methods"
+	AllowlistTypeOrigins    AllowlistType = "origins"
+	AllowlistTypeServices   AllowlistType = "services"
+	AllowlistTypeUserAgents AllowlistType = "user_agents"
+)
+
+type UserApp struct {
+	ID                  UserAppID
+	AccountID           AccountID
+	PlanType            string
```

Comment:
Thoughts on creating a type for plantype?

Reply:
I like it and I have done it. :)